lib/code_corps/map_utils.ex

```diff
@@ -8,18 +8,18 @@ defmodule CodeCorps.MapUtils do
   def keys_to_string(map), do: stringify_keys(map)
 
   # Goes through a list and stringifies keys of any map member
-  def stringify_keys(nil), do: nil
-  def stringify_keys(%DateTime{} = val), do: val
-  def stringify_keys(map = %{}) do
+  defp stringify_keys(nil), do: nil
+  defp stringify_keys(%DateTime{} = val), do: val
```

Comment:
I'm not sure if there's an `is_struct` guard clause, so maybe we could instead change line 13 to `when is_map(map) and not is_struct(map)`, but without something like that, this has to stay.